status_history.go

```diff
@@ -0,0 +1,21 @@
+package braintree
+
+import (
+	"time"
+)
+
+// StatusHistory contains information about the last 50 timestamps where
+// something changed about the subscription.
+type StatusHistory struct {
+	StatusEvents []StatusEvent `xml:"status-event"`
+}
+
+// StatusEvent contains information about what and when something
+// changed about the subscription.
+type StatusEvent struct {
```

Comment:
Lets call these types `SubscriptionStatusHistory` and `SubscriptionStatusEvent` because there are also status events for transactions that haven't been added to the this SDK yet, and they have different fields so it would be difficult to share this type with transactions.

Reply:
Ok, I will change the filename accordingly.